org.freedesktop.Sdk.Extension.openjdk17.yaml

```diff
@@ -0,0 +1,170 @@
+---
+id: org.freedesktop.Sdk.Extension.openjdk17
+branch: '21.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+build-extension: true
+sdk: org.freedesktop.Sdk
+separate-locales: false
+appstream-compose: false
+cleanup:
+  - /share/info
+  - /share/man
+build-options:
+  no-debuginfo: true
+  strip: true
+  prefix: /usr/lib/sdk/openjdk17
+  env:
+    V: '1'
+modules:
+  - name: bootstrap_jdk
+    buildsystem: simple
+    cleanup:
+      - '*'
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://fedorapeople.org/~mbooth/bootstrap_jdk/bootstrap-openjdk-16.0.1.0.9-1.rolling.fc32.x86_64.tar.bz2
+        dest-filename: java-openjdk.tar.bz2
+        sha512: 0084f99558a73788438df4fe62700584b41ee962d7de7bd322d8950207c88acfb3839c788def906926325f782ae86068315352dfc3b8c6d5375dec192cf3adeb
+      - type: file
+        only-arches:
+          - arm
+        url: https://fedorapeople.org/~mbooth/bootstrap_jdk/bootstrap-openjdk-16.0.1.0.9-1.rolling.fc32.armv7hl.tar.bz2
+        dest-filename: java-openjdk.tar.bz2
+        sha512: 9fad9ef72174ad268ecf0d85b0eef8a6ee54c68de4060947a9bf2593c21a9cf0bc119c068cde67163b402cc2da972a79a7f1d7875d63d6899dcffd060689fa08
```

Comment:
this is unecessary armv7 is no longer supported in either 20.08 or 21.08

Reply:
Yep, I note that 19.08 is now EOL :-)